tools/clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -12334,7 +12334,7 @@ static int ValidateAttributeIntArg(Sema &S, const AttributeList &Attr,
     } else {
       if (ArgNum.isInt()) {
         value = ArgNum.getInt().getSExtValue();
-        if (!(E->getType()->isIntegralType(S.Context)) || value < 0) {
+        if (!(E->getType()->isIntegralOrEnumerationType()) || value < 0) {
```

Comment:
Looks like you picked up some unexpected changes.

Reply:
I think I accidentally pulled from the main branch. should be fixed now